VocalTractLabApi.sln

```diff
@@ -24,11 +24,9 @@ Global
 		{A9B93E13-A27D-42B6-BE82-7B564A6071BA}.Release|x86.ActiveCfg = Release|Win32
 		{A9B93E13-A27D-42B6-BE82-7B564A6071BA}.Release|x86.Build.0 = Release|Win32
 		{59651411-0530-4487-ACD6-BF141C2DFCCF}.Debug|x64.ActiveCfg = Debug|x64
-		{59651411-0530-4487-ACD6-BF141C2DFCCF}.Debug|x64.Build.0 = Debug|x64
```

Comment:
What is this and why was it removed?

Reply:
I did not edit the file, must have been some automatic procedure by Visual Studio?

Reply:
Probably. Should be safe to ignore.